wasm/jsapi/module/moduleSource.any.js

```diff
@@ -0,0 +1,37 @@
+// META: global=window,dedicatedworker,jsshell,shadowrealm
+// META: script=/wasm/jsapi/wasm-module-builder.js
+// META: script=/wasm/jsapi/assertions.js
+
+let emptyModuleBinary;
+setup(() => {
+  emptyModuleBinary = new WasmModuleBuilder().toBuffer();
+});
+
+test(() => {
+  assert_equals(typeof AbstractModuleSource, "undefined");
+  const AbstractModuleSource = Object.getPrototypeOf(WebAssembly.Module).constructor;
```

Comment:
Isn't `AbstractModuleSource` itself `Object.getPrototypeOf(WebAssembly.Module)` itself, and `.constructor` is the `Function` constructor?

Or at least, that's what I see when doing `Object.getPrototypeOf(class X extends class Y {} {})`.

```suggestion
  const AbstractModuleSource = Object.getPrototypeOf(WebAssembly.Module);
```

Reply:
Yes, you're right, thank you.